src/validation.h

```diff
@@ -204,7 +205,8 @@ void StopScriptCheckWorkerThreads();
 CTransactionRef GetTransaction(const CBlockIndex* const block_index, const CTxMemPool* const mempool, const uint256& hash, const Consensus::Params& consensusParams, uint256& hashBlock);
 
 double ConvertBitsToDouble(unsigned int nBits);
-CAmount GetBlockSubsidy(int nBits, int nHeight, const Consensus::Params& consensusParams, bool fSuperblockPartOnly = false);
+CAmount GetBlockSubsidyBase(int nPrevBits, int nPrevHeight);
```

Comment:
It was used to avoid passing `consensusParams` when backporting coinstats index (now split into https://github.com/dashpay/dash/pull/5501), a different approach was eventually taken. Commit dropped.